studio/runner.py

```diff
@@ -278,6 +340,7 @@ def main(args=sys.argv):
     else:
         if queue_name == 'local':
             queue = LocalQueue()
+            queue.clean()
```

Comment:
Like Karl pointed out, I think we should not clean the local queue before running the experiments. We can add a command that does it to the CLI; but generally the experiment ends up in a local queue unprocessed is a sign that something went wrong in the runner (i.e. when local_worker works correctly, this should not be happening), so I'd rather fix the problems there.

Reply:
There is a bug where the code crashes if the folders in ~/studioml/experiments remain, but the corresponding file in the local queue is deleted. What would you suggest to do to fix this issue?

Reply:
Could you provide an example (name of the branch + what command are you running)? In principle, data in ~/.studioml/experiments is just cache of experiment data; it should not be deleted when the queue is cleaned.

Reply:
we will leave in queue.clean() for now, will address this in a future PR